### DISCUSSION TITLE
containers are created in lowercase

### DISCUSSION THREAD

**Discussion by pisabird** [tteck/Proxmox#2001] [General]
The containers are created in lowercase. Your output for the hostname is in lowercase only, even though uppercase was entered.

**Comment by tteck** [comment]
Using lowercase letters for hostnames is a recommended best practice, as it helps maintain consistency and minimizes the likelihood of confusion. The scripts in use adhere to this practice.

**Reply by pisabird** [reply]
well noted, like it's not a bug it's a feature

**Reply by tteck** [reply]
Correct
